### PR TITLE
Issue 211 : Remove toggle when no baseline or no diff.

### PR DESCRIPTION
**Pull request by suratdas**
Fixes issue : https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/211

### CODE REVIEW

File: src/components/TestDetailsModal.tsx [hunk 1]
```diff
@@ -153,6 +153,7 @@ const TestDetailsModal: React.FunctionComponent<{
 
   useHotkeys("d", () => setIsDiffShown((isDiffShown) => !isDiffShown));
   useHotkeys("ESC", () => handleClose());
+  const shouldDiffHotKeyBeActive = !!testRun.baselineName && !!testRun.diffName;
```
[the remark this answers is not in the document]
Comment (suratdas) [reply src/components/TestDetailsModal.tsx:156]: The reason I put it there is - when there is no baseline (first-time case), there is nothing to show as diff. So, we may not need that toggle button.

File: src/components/TestDetailsModal.tsx [hunk 2]
```diff
@@ -335,7 +338,7 @@ const TestDetailsModal: React.FunctionComponent<{
             />
           </Grid>
           <Grid item xs={6} className={classes.drawAreaItem}>
-            {isDiffShown ? (
+            {(isDiffShown && shouldDiffHotKeyBeActive) ? (
```
[the remark this answers is not in the document]
Comment (suratdas) [reply src/components/TestDetailsModal.tsx:341]: isDiffShown rightly controls whether the toggle should be displayed or not. But even when the toggle is not displayed hotkey (D) was active, that is why I added another variable to disable the hotkey if the toggle is not displayed.

File: src/components/TestDetailsModal.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (pashidlos) [reply src/components/TestDetailsModal.tsx:156]: I guess it's not possible to have `diffName` but not `baselineName`

File: src/components/TestDetailsModal.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pashidlos) [reply src/components/TestDetailsModal.tsx:341]: now I got it 
the thing is that key is still active 
you could disable it by adding condition inside `useHotkeys`
```
  useHotkeys("d", () => shouldDiffHotKeyBeActive  && setIsDiffShown((isDiffShown) => !isDiffShown));
```

[the remark this answers is not in the document]
Comment (suratdas) [reply src/components/TestDetailsModal.tsx]: Thanks. Fixed.

File: src/components/TestDetailsModal.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (suratdas) [reply src/components/TestDetailsModal.tsx]: Thanks. Fixed.

**Review by pashidlos — APPROVED** [review]
LGTM!

**Comment by pashidlos** [comment]
@suratdas thanks for contribution!

**Pull request merged**